src/VisualStudio/CSharp/Impl/Options/AutomationObject/AutomationObject.Formatting.cs

```diff
@@ -0,0 +1,340 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Formatting;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.Options
+{
+    public partial class AutomationObject
+    {
+        public int Indent_BlockContents
+        {
+            get { return GetBooleanOption(CSharpFormattingOptions2.IndentBlock); }
+            set { SetBooleanOption(CSharpFormattingOptions2.IndentBlock, value); }
+        }
+
+        public int Indent_Braces
+        {
+            get { return GetBooleanOption(CSharpFormattingOptions2.IndentBraces); }
+            set { SetBooleanOption(CSharpFormattingOptions2.IndentBraces, value); }
+        }
+
+        public int Indent_CaseContents
+        {
+            get { return GetBooleanOption(CSharpFormattingOptions2.IndentSwitchCaseSection); }
+            set { SetBooleanOption(CSharpFormattingOptions2.IndentSwitchCaseSection, value); }
+        }
+
+        public int Indent_CaseContentsWhenBlock
+        {
+            get { return GetBooleanOption(CSharpFormattingOptions2.IndentSwitchCaseSectionWhenBlock); }
+            set { SetBooleanOption(CSharpFormattingOptions2.IndentSwitchCaseSectionWhenBlock, value); }
+        }
+
+        public int Indent_CaseLabels
+        {
+            get { return GetBooleanOption(CSharpFormattingOptions2.IndentSwitchSection); }
+            set { SetBooleanOption(CSharpFormattingOptions2.IndentSwitchSection, value); }
+        }
+
+        public int Indent_FlushLabelsLeft
+        {
+            get
+            {
+                var option = _workspace.Options.GetOption(CSharpFormattingOptions2.LabelPositioning);
+                return option == LabelPositionOptions.LeftMost ? 1 : 0;
+            }
+
+            set
+            {
+                _workspace.TryApplyChanges(_workspace.CurrentSolution.WithOptions(_workspace.Options
+                    .WithChangedOption(CSharpFormattingOptions2.LabelPositioning, value == 1 ? LabelPositionOptions.LeftMost : LabelPositionOptions.NoIndent)));
```

Comment:
make helper for this common pattern?  can then trim down these props to be more in line with the others?